src/tools/create-require.js

```diff
@@ -0,0 +1,21 @@
+'use-strict';
+
+const { dependencies, devDependencies } = require('../../package.json');
+const { RequireModuleError } = require('../errors');
+
+const createRequire = () => module => {
+  const libs = Object.keys(dependencies).concat(Object.keys(devDependencies));
```

Comment:
If an app doesn't list any `devDependencies`, `Object.keys(undefined)` throws an error. Also, I don't believe godzilla apps have any devDeps.

Also, we might want to think about a future where apps don't have to explicitly depend on `core` (if we use lambda layers). I'd import the name from `constants` and list it explicitly in `allowed`

Reply:
I'm all for that. Again, just going right to `require` makes this code moot anyway.